.travis.yml

```diff
@@ -1,39 +1,29 @@
 language: julia
-
 os:
-  - osx
   - linux
-
+  - osx
+  - windows
 julia:
-  - 1.0
-  - 1.1
   - 1.2
   - nightly
-
-# # Uncomment the following lines to allow failures on nightly julia
-# # (tests will run but not make your overall status red)
-# matrix:
-#   allow_failures:
-#   - julia: nightly
-
 notifications:
   email: false
-
-#script: # the default script is equivalent to the following
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia --project -e 'using Pkg; Pkg.build(); Pkg.test(; coverage=true)';
-
-after_success:
-  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
-#  - julia -e 'using Pkg; Pkg.add("Coverage"); using Coverage; Codecov.submit(Codecov.process_folder())';
```

Comment:
Why remove coverage?

Reply:
Is no longer needed. Missed adding the line `codecov: true` which is the updated way.

Reply:
I don't know if it's still the case, but for a while `codecov: true`/`coveralls: true` caused issues with how builds report failure (as errors instead of failures).

Reply:
Would be nice to check that since Coverage.jl has their documentation indicating that method currently.